rules/follow-naming-conventions-for-your-boolean-property/rule.md

```diff
@@ -36,4 +36,22 @@ Good Example - Naming Convention for Boolean Property
 
 :::
 
-We have a program called [SSW Code Auditor](https://www.ssw.com.au/ssw/CodeAuditor/) to check for this rule.
\ No newline at end of file
+### **Naming Boolean state Variables in Frontend code**
+
+When it comes to state management in frameworks like Angular or React, a similar principle applies, but with a focus on the continuity of the action.
+
+ For instance, if you are tracking a process or a loading state, the variable should reflect the ongoing nature of these actions. Instead of "isLoaded" or "isProcessed," which suggest a completed state, use names like "isLoading" or "isProcessing."
+
+These names start as false, change to true while the process is ongoing, and revert to false once completed.
+
+```csharp
+const [isLoading, setIsLoading] = useState(false); // Initial state: not loading
+```
```

Comment:
```suggestion
```js
const [isLoading, setIsLoading] = useState(false); // Initial state: not loading
```
```

Reply:
Done - changes to js.